README.md

```diff
@@ -6,7 +6,7 @@ our [support forum](http://gatkforums.broadinstitute.org/wdl).
 
 * [Draft 1 Language Specification](https://github.com/openwdl/wdl/blob/master/versions/draft-1/SPEC.md) (closed)
 * [Draft 2 Language Specification](https://github.com/openwdl/wdl/blob/master/versions/draft-2/SPEC.md) (closed)
-* [Draft 3 Language Specification](https://github.com/openwdl/wdl/blob/master/versions/draft-3/SPEC.md) (open)
+* [Draft 3 Language Specification](versions/draft-3/SPEC.md) (open)
```

Comment:
I added this because trying to view my change in place brought me back to `openwdl/master` rather than the local change. Just a QoL improvement for contributors!